src/samples/Scoping/index.html

```diff
@@ -0,0 +1,61 @@
+<!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
+
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Fabric's Version Scoping Test Page</title>
+
+  <!-- Fabric core and Components CSS -->
+  <link rel="stylesheet" type="text/css" href="../../../css/fabric.css">
+  <link rel="stylesheet" type="text/css" href="../../../css/fabric-4.scoped.css">
+  <link rel="stylesheet" type="text/css" href="../../../css/fabric-5.mock.scoped.css">
+
+  <style type="text/css">
+    html,
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    h1 {
+      margin: 0;
+      padding: 10px;
+      margin-bottom: 20px;
+      padding-left: 25px;
+    }
+
+    .Card {
+      min-height: 10em;
+      margin-bottom: 2em;
+      padding: 20px;
+    }
+
+    section {
+      margin-bottom: 20px;
+    }
+  </style>
+</head>
+
+<body>
+  <header>
+    <h1 class="ms-font-su ms-fontColor-white ms-bgColor-themeDarker">Version Scoping Sample</h1>
+  </header>
+  <main class="ms-Grid">
+    <div class="ms-Grid-row">
+      <div class="ms-Grid-col ms-u-sm10 ms-u-smPush1 ms-u-smPull1">
+        <h2 class="ms-font-xxl">Version 4</h2>
+      </div>
+    </div>
+
+    <div class="ms-Grid-row">
+      <div class="ms-Grid-col ms-u-sm10 ms-u-smPush1 ms-u-smPull1">
+        <h2 class="ms-font-xxl">Version 4</h2>
```

Comment:
I'm not seeing the scope classes in use on this page. Did you want to show an example of how v4 and v5 could coexist with different styles?